magicbot/magicrobot.py

```diff
@@ -23,7 +23,7 @@ class MagicInjectError(ValueError):
     pass
 
 
-class MagicRobot(wpilib.SampleRobot):
+class MagicRobot(wpilib.RobotBase):
```

Comment:
This all seems fine, but I'm not sure if we can inherit from RobotBase. Once the next wpilibc comes out we can fix it, but I seem to recall an issue with the protected constructor.

Reply:
Well, our only other option is to subclass TimedRobot, which would require a significant rewrite.